using-mobile-extensions/mobile-core/signals/signals-extension-and-rules-engine-integration.md

```diff
@@ -2,55 +2,90 @@
 
 The rules that you set up can use the available triggers and conditions, which results in one of the following actions:
 
-* A postback
-* PII data
-* An open URL request
+* Send Postback
+* Send PII
+* Open URL
 
 After these actions have been configured to be triggered and published, the Signal extension carries out the requested actions.
 
 To send PII data to external destinations, the `PII` action can trigger the Rules engine when certain triggers and traits match. When setting a rule, you can also set the `PII` action for a Signal event. The `collectPii` API can then be used to trigger the rule and send the PII data.
 
 ## Rules tokens <a id="rules-tokens"></a>
 
-Tokens are special strings used in rule actions as values, which are expanded by the SDK when the action is carried out. The format of a token is \`
+Tokens are special strings used in rule actions as values, which are expanded by the SDK when the action is carried out. The format of a token is `{%%token%%}`, where token is any data element defined in Adobe Launch for a mobile property which identifies the source of the data from which the token is expanded. For example, `{%%My Data element for ECID%%}` can be used in the Signal Post back action, where `My Data element for ECID` is a Data element created using the Mobile Core extension, and Data element type Experience Cloud ID.
 
-\`, which is any period-separated string that identifies the source of the data from which the token is expanded. It can also be one of the reserved key names as described in the [Matching and Retrieving Values by keys](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine#matching-and-retrieving-values-by-keys).
+The token can also be one of the reserved key names as described in the [Matching and Retrieving Values by keys](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-details#matching-and-retrieving-values-by-keys).
```

Comment:
- Please add a period after "names". 

- For the 2nd sentence, please reword as follows:
"For more information, see [Matching and Retrieving Values by keys](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-details#matching-and-retrieving-values-by-keys)."